docs/source/notebooks/finch-usage.ipynb

```diff
@@ -47,15 +47,19 @@
      "text": [
       "Help on method frost_days in module birdy.client.base:\n",
       "\n",
-      "frost_days(tasmin=None, freq='YS') method of birdy.client.base.WPSClient instance\n",
+      "frost_days(tasmin=None, missing_options=None, check_missing='any', freq='YS') method of birdy.client.base.WPSClient instance\n",
```

Comment:
This change breaks Jenkins (against Finch deployed on PAVICS).  Can we have a new release of Finch to deploy to PAVICS?

Reply:
Yes, see https://github.com/bird-house/finch/pull/138